agent/metrics_test.go

```diff
@@ -81,3 +79,17 @@ func (s *MetricsTestSuite) TestPersistSystemStats() {
 
 	s.True(s.comm.GetSystemInfoLength() >= 1)
 }
+
+// TestMetricsOnWindows adds a longer delay to ensure that we capture something
+// on Windows. On Windows we run only this test.
+func (s *MetricsTestSuite) TestMetricsOnWindows() {
```

Comment:
This is run regardless of the OS I think? Though wouldn't it be cleaner to define a different sleep interval on Windows rather than a new test?

Reply:
Done